### ISSUE TITLE
Switch to docs.asciidoctor.org domain

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/docs.asciidoctor.org#14]
Currently, this site is being deployed to a temporary domain on Netlify. The Netlify domain should only be used for deploy previews. The main site should run on docs.asciidoctor.org. Since this domain is not currently in use, we can provide early access to it, even before asciidoctor.org/docs redirects to it.

**Comment by mojavelinux** [comment]
I added docs.asciidoctor.org to the DNS zone file and pointed it at Netlify (using an A DNS record).

I also set up Travis CI to trigger the Netlify build daily. (This was previously being done by the asciidoctor.js build, but that can now be removed. cc: @Mogztter)

All that's left is to add the custom domain in the Netlify configuration, click through to enable HTTPS, and force HTTPS. That takes about 5 minutes. Then, docs.asciidoctor.org will be live. I'll hold off on doing that until the content is ready.
